index.js

```diff
@@ -1 +1,17 @@
-export * from './dist/auro-header';
+import { AuroHeader } from './src/auro-header';
+
+/**
+ * Register Custom Element.
+ * @param {Object} name - Name to use for custom element.
+ * @returns {void}
+ */
+ const registerComponent = (name = 'custom-header') => {
+  // alias definition
+  if (!customElements.get(name)) {
+    customElements.define(name, class extends AuroHeader {});
+  }
+}
+
+registerComponent('auro-header');
+
+export { registerComponent }
```

Comment:
There are newly discovered issues with this solution. @Izook I am curious as to why this issue was worked on while the issue is marked as `not-reviewed` and is currently in the refinement status?

Reply:
Sorry, I did not realize that the issue was in the `non-reviewed` state. I believed that with my communication of the change with Jason and Brandon the change was ready to be worked on. I am happy to wait for Auro to review the issue. Should I hold off on this until both issues have been marked as `reviewed`?

Reply:
You updated the index.js to register the comment, but the code for component registration is still in the ./src/auro-header.js file. 

That needs to be removed.

Reply:
Once that code is removed, the test docs need to be updated to point to the index.js at the root. 

Line 29 of the rollup config needs to be updated too. 
`input: 'src/auro-header.js',`

Reply:
Updated `index.js`, tests, and rollup